test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/ChatCompletionRequestMessage.cs

```diff
@@ -53,6 +53,6 @@ internal ChatCompletionRequestMessage(ChatCompletionRequestMessageRole role, str
         /// </summary>
         public string Name { get; set; }
         /// <summary> The name and arguments of a function that should be called, as generated by the model. </summary>
-        public CreateFunctionCall FunctionCall { get; set; }
+        public ChatCompletionRequestMessageFunctionCall FunctionCall { get; set; }
```

Comment:
This is an anonymous model: https://github.com/Azure/autorest.csharp/blob/37812dab10bf294ef00614f615e6a8df03f46100/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L225